include_physics/ideal_MHD.h

```diff
@@ -388,8 +388,9 @@ inline void init_fields(uint ix, uint iy, uint iz, global REAL* u) {
 //--------------------------------------------------------------------------------------------------
 
 
-#ifndef USE_PERIODIC
+#if !(defined(USE_PERIODIC_X) && defined(USE_PERIODIC_Y) && defined(USE_PERIODIC_Z))
 
+#define NONPERIODIC_BOUNDARY_EXISTS
```

Comment:
This constant can be useful in other files, too. Could you please move the definition to another file that is included by default (e.g. utils.h)?